tools/myst/enc/clock.c

```diff
@@ -16,17 +16,26 @@ static long enc_clock_res = 0;
 
 int myst_setup_clock(struct clock_ctrl* ctrl)
 {
+    // ctrl is a host address.
+    // Store a copy of its content in enclave address to
+    // avoid reading host memory.
+    struct clock_ctrl* ctrl_enclave = NULL;
     int ret = -1;
     if (ctrl != NULL)
     {
         if (!oe_is_outside_enclave(ctrl, sizeof(struct clock_ctrl)))
             return ret;
 
+        // Make a copy of ctrl
+        if (!(ctrl_enclave = malloc(sizeof(struct clock_ctrl))))
+            return -ENOMEM;
+        memcpy(ctrl_enclave, ctrl, sizeof(struct clock_ctrl));
```

Comment:
Ideally, we should ensure using the version of memcpy that only does 8-byte mov.

Reply:
Do you suggest we use `oe_memcpy_aligned` here? I think that version is not yet released?

Reply:
Although the PR that provides `oe_memcpy_aligned` has been merged, Mystikos is not using that version of OpenEnclave. Shall we update the OE version used in Mystkos?